suites/quincy/nvmeof/tier-1_nvmeof_functional_Regression.yaml

```diff
@@ -108,24 +108,24 @@ tests:
         rep-pool-only: true
         rep_pool_config:
           pool: rbd
-        install: false                           # Run SPDK with all pre-requisites
-        subsystems: # Configure subsystems with all sub-entities
+        install: true                           # Run SPDK with all pre-requisites
```

Comment:
@preethisn IMO, this should be false. Please clarify. With "install being True" here again for node6, test again clones git repo, run SPDK and spin GW on same node6 which was done earlier with Data Integrity test above.

Reply:
@rahullepakshi This is first test case in the test suit so install needs to be True. We have re-arranged the tests in the functional suit as reboot of the client changes the block device id when more subsystems based luns are present

Reply:
@preethisn thanks for the information. Seems like I need more understanding on this, I will sync up with you later.

Reply:
seems like install is changed to false as requested which is second test on same node